task_20/dima.gres/src/main/java/com/dima/District.java

```diff
@@ -0,0 +1,18 @@
+package com.dima;
+
+import java.io.Serializable;
+import java.util.Comparator;
+
+public class District implements Comparator<Student>, Serializable {
+  @Override
+  public int compare(Student tmp, Student tmp1) {
+    if (tmp.getCity().getDistrict().compareTo(tmp1.getCity().getDistrict()) < 0) {
```

Comment:
You can use:
return tmp.getCity().getDistrict().compareTo(tmp1.getCity().getDistrict());